openedx/core/djangoapps/cache_toolbox/middleware.py

```diff
@@ -88,7 +88,7 @@
 from django.utils.crypto import constant_time_compare
 from django.utils.deprecation import MiddlewareMixin
 
-from openedx.core.djangoapps.safe_sessions.middleware import SafeSessionMiddleware
+from openedx.core.djangoapps.safe_sessions.middleware import SafeSessionMiddleware, _mark_cookie_for_deletion
```

Comment:
It would be nice to add SafeSessionMiddleware to the docstring of this module, and to mention that the dependency between these two middleware.

Reply:
Thanks @robrap, I have added the docstring.